xonsh/ptk_shell/formatter.py

```diff
@@ -20,6 +20,7 @@ def __call__(
         fields=None,
         threaded=False,
         prompt_name: str = None,
+        **_
```

Comment:
What it is purpose of this?

Reply:
to match the signature - mypy was giving error before.

Reply:
Which signature? Of the same method on the parent?

Reply:
Yes. It has to have same signature.

Reply:
According to mypy? That isn't generally true in Python

Reply:
Sorry I am pretty new to mypy because I have heard about how often it breaks :wink:

Reply:
Yes it adds some kind of rigidity to the dynamic ways. But I think they help bigger projects.

Reply:
so closing it then. I am collapsing as I finish each. Please feel free to check and raise any queries.